docs/workflows/_sdk-setup/_authentication-swift.mdx

```diff
@@ -0,0 +1,22 @@
+The `authenticate` function expects a `URL` and a `PasskeyID`. The Beyond Identity specific URL is generated during the OAuth2 authorization flow and carries with it a JWT that contains information specific to the current authorization request. When passing this URL into the `authenticate` function, this will perform a challenge/response against the private key bound to the passkey on your device. This function should be used in conjunction with [isAuthenticateUrl](#authenticate-url-validation) in order to determine if the URL being passed in is a valid authenticate URL.
+
+Before calling this function you will need to ask the user to select a passkey that has been bound to the device. A selection view can be built in conjunction with [getPasskeys](#listing-passkeys). 
+
+#### Usage
+
+```javascript
+Embedded.shared.authenticate(
+    url: URL,
+    passkeyID: PasskeyID,
```

Comment:
for iOS this is just `id` because the Type is `Passkey.Id` so you can't pass any other id except a `Passkey.Id` :)
```suggestion
    id: Passkey.Id,
```